AlphaWallet/Localization/es.lproj/Localizable.strings

```diff
@@ -329,3 +329,5 @@
 "a.claim.token.failed.notEnoughXDAI.title" = "a.claim.token.failed.notEnoughXDAI.title";
 "photos" = "Photos";
 "a.claim.token.failed.serverDown" = "a.claim.token.failed.serverDown";
+
+"a.claim.token.wrongNetworkLink" = "a.claim.token.wrongNetworkLink";
```

Comment:
Remember the %@ in other Localisable.strings files :)

Reply:
It is referencing it directly, not using the text?

Reply:
Hmm.. Maybe it'd be harmless actually. If you switch to say `es` and access the R.generated method with an arg, does it still just display "a.claim.token.wrongNetworkLink" verbatim?

I was thinking of changing it to "a.claim.token.wrongNetworkLink %@", but it'd still look like garbage. So as long as it runs and doesn't crash, that's good.